mlflow/ml-package-versions.yml

```diff
@@ -454,3 +454,16 @@ paddle:
     maximum: "2.4.1"
     run: |
       pytest tests/paddle/test_paddle_autolog.py
+
+transformers:
+  package_info:
+    pip_release: "transformers"
+    install_dev: |
+      pip install git+https://github.com/huggingface/transformers
+  models:
+    minimum: "4.27.2"
+    maximum: "4.27.2"
```

Comment:
MLR 12.2 LTS includes transformers 4.25.1. Since this MLR is so new and is LTS, it would be great if our flavor were compatible with it. 

One thing I ran into with 4.25.1 is that `GraphormerPreTrainedModel` isn't defined:

```
from transformers import AutoTokenizer, AutoModelWithLMHead, pipeline

tokenizer = AutoTokenizer.from_pretrained('microsoft/DialoGPT-small')
model = AutoModelWithLMHead.from_pretrained('satvikag/chatbot')

import mlflow.transformers

mlflow.transformers.log_model(
   pipeline(task="conversational", model=model, tokenizer=tokenizer),
   artifact_path="model",
)
```

```
Traceback (most recent call last):
  File "chat.py", line 35, in <module>
    mlflow.transformers.log_model(
  File "/Users/corey.zumar/mlflow/mlflow/transformers.py", line 467, in log_model
    return Model.log(
  File "/Users/corey.zumar/mlflow/mlflow/models/model.py", line 547, in log
    flavor.save_model(path=local_path, mlflow_model=mlflow_model, **kwargs)
  File "/Users/corey.zumar/mlflow/mlflow/transformers.py", line 308, in save_model
    if _should_add_pyfunc_to_model(built_pipeline):
  File "/Users/corey.zumar/mlflow/mlflow/transformers.py", line 841, in _should_add_pyfunc_to_model
    transformers.GraphormerPreTrainedModel,
  File "/Users/corey.zumar/opt/anaconda3/envs/base2/lib/python3.8/site-packages/transformers/utils/import_utils.py", line 1086, in __getattr__
    raise AttributeError(f"module {self.__name__} has no attribute {name}")
AttributeError: module transformers has no attribute GraphormerPreTrainedModel
```

Reply:
Fixed this with a hasattr check